core/domain/stats_services.py

```diff
@@ -296,41 +296,35 @@ def update_exp_issues_for_new_exp_version(
         revert_to_version: int|None. If the change is a revert, the version.
             Otherwise, None.
     """
-    # TODO(pranavsid98): Convert below two lines to use get_by_id() instead.
-    exp_issues_model = stats_models.ExplorationIssuesModel.get_model(
-        exploration.id, exploration.version - 1)
-    if exp_issues_model is None:
+    exp_issues = get_exp_issues(exploration.id, exploration.version - 1)
+    if exp_issues is None:
         create_exp_issues_for_new_exploration(
             exploration.id, exploration.version - 1)
         return
-    exp_issues = get_exp_issues_from_model(exp_issues_model)
 
     # Handling reverts.
     if revert_to_version:
-        # TODO(pranavsid98): Convert below two lines to use get_by_id() instead.
-        old_exp_issues_instance = stats_models.ExplorationIssuesModel.get_model(
-            exploration.id, revert_to_version)
-        old_exp_issues = get_exp_issues_from_model(old_exp_issues_instance)
+        old_exp_issues = get_exp_issues(exploration.id, revert_to_version)
         exp_issues.unresolved_issues = old_exp_issues.unresolved_issues
         exp_issues.exp_version += 1
         create_exp_issues_model(exp_issues)
         return
 
     playthrough_ids_by_state_name = collections.defaultdict(list)
 
-    for idx, exp_issue in enumerate(exp_issues.unresolved_issues):
+    for e_idx, exp_issue in enumerate(exp_issues.unresolved_issues):
```

Comment:
nit: rename to `i_idx` since it's more important to note that the index is referring to an *issue*, not an *exploration*.

Reply:
Done.